setup.cfg

```diff
@@ -12,8 +12,8 @@ all_files  = 1
 [upload_sphinx]
 upload-dir = docs/build/html
 
-[pep8]
-ignore = E221
+[pycodestyle]
```

Comment:
At least pycodestyle 2.0.0 still reads from the [pep8] section, for compatibility we could just duplicate this section.